sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -397,7 +397,13 @@ public String getId() {
         if (queryInfo != null && queryInfo.hasSelectValue()) {
             List<T> transformedResults = response.getResults()
                                              .stream()
-                                             .map(d -> d.get(Constants.Properties.VALUE))
+                                             .map(d -> {
+                                                 if (d.has(Constants.Properties.VALUE)) {
+                                                     return d.get(Constants.Properties.VALUE);
```

Comment:
Don't return in an if else. Predeclare the variablee and return at the end.

Reply:
changed